logging/google/cloud/logging/handlers/app_engine.py

```diff
@@ -67,3 +69,15 @@ def get_gae_resource(self):
             },
         )
         return gae_resource
+
+    def get_gae_labels(self):
+        """Return the labels for GAE app which includes trace_id.
+
+        :rtype: dict
+        :returns: Labels for GAE app.
+        """
+        trace_id = get_trace_id()
+        gae_labels = {
```

Comment:
So with this code we still end up sending a trace ID of `'None'` in the labels, I think I would prefer us to just omit the label if the trace ID isn't available, so:

```python
gae_labels = {}

trace_id = get_trace_id()
if trace_id is not None:
    gae_labels[...] = trace_id

return gae_labels
```